roles/kube_lvm_storage/meta/main.yml

```diff
@@ -1,2 +1,3 @@
 dependencies:
-    - role: kubespray_module
+  - role: kubespray_module
+  - role: setup_lvm_lv
```

Comment:
Dependencies like that make it implicit, ( and often explicit is better than implicit) and then difficult to change. I tend to not have dependency if not strictly needed (here we can setup kube_lv if someone expose the same API, we do not need to be setup_lvm_lv specifically)